tests/test_octodns_provider_transip.py

```diff
@@ -134,6 +138,9 @@ def make_expected(self):
         return expected
 
     def test_init(self):
+        if self.bypass_tests:
```

Comment:
See https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures for an official way to do this.

Reply:
Done.

Reply:
I still needed to omit transip to get a 100% coverage.